### DISCUSSION TITLE
Need help adding a stance for crawling.

### DISCUSSION THREAD

**Discussion by Mykeuaa16** [Sixze/ALS-Refactored#553] [Q&A]
I got all my animations working and I set up all my anim BPs but I can't find a few things like where to add the CrawlState.StrideBlend or how to set the stance. I tried set "desired stance" but that didn't work. Any suggestions or examples would be amazing! There really does need to be a documentation for this project to help people use it. is there a discord server?

**Comment by xkamronx** [comment]
I had to modify the C++ to add additional stances, along with creating a new function to force the desired stance. The base code doesn't account for new stance tags, so Desired Stance doesn't do what you'd expect without digging in to the C++.
